wordpress-mobile/WordPress-iOS
# [SIWA] Handle credential state changes

Ref #12399 
Related WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/139

This uses the WPAuth changes to:
- Accept the `CredentialState` provided by WPAuth.
- If the user id is `revoked`, log out of the account.

There should be no visible difference from the previous implementation.

To test:

This should be tested on a real device for proper keychain handling.

- Log in / sign up with Apple.
- Background the app.
- Disconnect in your Apple account on the device.
  - Go to iOS Settings.
  - Select your iCloud account at the top.
  - Go to Password & Security > Apps Using Your Apple ID.
  - Select WordPress > Stop Using Apple ID.
- Open the app.
- The app should log you out.

Update release notes:
